src/patch.jl

```diff
@@ -49,6 +49,18 @@ function Base.:(==)(pe1::PatchEnv, pe2::PatchEnv)
     return pe1.mapping == pe2.mapping && pe1.debug == pe2.debug
 end
 
+if VERSION < v"1.5"
+    # A version of `mergewith` suitable for use in `merge` below.
+    function mergewith(combine, d1::AbstractDict, d2::AbstractDict)
```

Comment:
This seemed like the least bad solution — can delete this if/when we decide to only support sufficiently new Julia versions.

Reply:
[Compat.jl](https://github.com/JuliaLang/Compat.jl) has you covered.
```julia
using Compat: mergewith
```
Just need to use at least Compat 3.9.0

Reply:
Ah, that's much nicer, thanks for the tip!